packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -1,7 +1,6 @@
 //! The Axum webserver for accessing the Graph API datastore operations.
 //!
-//! Handler methods are grouped by routes that make up the `web_api`. See [`data_type`] for a
-//! grouped router example
+//! Handler methods are grouped by routes that make up the `web_api`.
```

Comment:
Oops can we drive-by this. It's not the `web_api` anymore but actually the REST one.

Reply:
```suggestion
//! Handler methods are grouped by routes that make up the REST API.
```